debian/pt-os-web-portal.postinst

```diff
@@ -44,13 +44,27 @@ case "$1" in
 		/usr/lib/pt-os-web-portal/handle-service-restart-in-policy add NetworkManager.service
 	fi
 
-  ;;
+	LAST_UPDATE_FOLDER="/var/lib/pt-os-web-portal"
+	STATE_FILE="${LAST_UPDATE_FOLDER}/state.cfg"
 
-\
-  abort-upgrade | abort-remove | abort-deconfigure | try-restart | triggered) ;;
+ 	# if upgrading from a previous version...
+	if [ ! -z "$2" ]; then
+		# and the state file doesn't exist (probably upgrading during onboarding)
```

Comment:
include in the comment that the web portal usually doesn't have a state file unless it's already been run

Reply:
eg there is an assumption that old versions of web portal don't create a state file before the update step